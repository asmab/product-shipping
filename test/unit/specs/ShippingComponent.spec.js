import Vue from 'vue'
import { shallowMount } from '@vue/test-utils'
import ShippingComponent from '@/components/ShippingComponent'

describe('Shipping component', () => {
    let wrapper

    beforeEach(() => {
        wrapper = shallowMount(ShippingComponent)
    })

    it('Shipping component looks as expected', () => {
        expect(wrapper.html()).toMatchSnapshot()
    })
})
