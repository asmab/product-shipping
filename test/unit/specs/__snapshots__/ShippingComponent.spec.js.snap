// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shipping component Shipping component looks as expected 1`] = `
<div class="main-content">
    <div class="tab-wrap">
        <input type="radio" name="tabs" id="tab1" checked="checked">
        <div id="tab1-content" class="tab-label-content">
            <label for="tab1"><span class="details-wrapper">
            Cheapest<br> <span class="cost"> 62.98 </span><span class="currency"> sek</span></span>
            </label>
            <div class="tab-content">
                <productlist-stub sorteditems="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]"></productlist-stub>
            </div>
        </div>
        <input type="radio" name="tabs" id="tab2">
        <div id="tab2-content" class="tab-label-content">
            <label for="tab2"><span class="details-wrapper">
            Fastest<br> <span class="days"> 1 </span><span class="info"> day</span></span>
            </label>
            <div class="tab-content">
                <productlist-stub sorteditems="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]"></productlist-stub>
            </div>
        </div>
        <input type="radio" name="tabs" id="tab3">
        <div id="tab3-content" class="tab-label-content">
            <label for="tab3"><span class="details-wrapper">
            Best<br> <span class="days"> 62.98<span class="currency"> sek</span> / </span> <span class="days"> 2 days </span></span>
            </label>
            <div class="tab-content">
                <productlist-stub sorteditems="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]"></productlist-stub>
            </div>
        </div>
        <div class="slide"></div>
    </div>
</div>
`;
